packages/analytics-browser/test/browser-client.test.ts

```diff
@@ -839,13 +839,16 @@ describe('browser-client', () => {
       expect(extendedLastEventTime > eventTime1).toBeTruthy();
 
       // send another event just before session expires (again)
+      // Mock Date.now() because isNewSession() depends on it
+      const dateNowMocked = jest.spyOn(Date, 'now').mockImplementation(() => extendedLastEventTime + 15);
```

Comment:
Is the +15 using the same value from L831? If so does it make sense for this to be put into a variable rather than using a magic number? Perhaps something like `sessionExtensionDelay`
Or changing `sessionTimeout` to a variable that can be used with +/- 1 as necessary

Reply:
It is and a good suggestion. But I'd prefer to leave it unchanged since it works as-is and is not hard to understand.

Reply:
Another reason is that this PR aims to fix the flakiness rather than to refactor.